### DISCUSSION TITLE
Safari 15 on iPhone/iPad compatibility

### DISCUSSION THREAD

**Discussion by Kris-B** [visgl/deck.gl#6778] [Q&A]
Hey,
I'm facing important rendering issues on safari 15 but only on iphone or ipad.
Users using my map reported the problem, but as I don't have any apple device, I'm testing with BrowserStack.

Screenshot on safari 14 iphone/ipad:

![safari_14a](https://user-images.githubusercontent.com/4072515/160180493-43944660-d4bf-4516-86cc-82faeae3a2ef.png)

Screenshot on safari 15 iphone/ipad:

![safari_15a](https://user-images.githubusercontent.com/4072515/160180560-f3d6b431-ec48-4df7-91f8-11053e836f72.png)

Even with very limited data and layers:
![safari_15b](https://user-images.githubusercontent.com/4072515/160180687-71ad9c95-2bc6-4958-9159-436326068ce9.png)

The map is hosted here: https://urbanicc.fr/index.php/carte-interactive

Any help would be very appreciated.

**Comment by Pessimistress** [comment]
Do you see this issue on any of the examples on https://deck.gl/examples?

**Reply by Kris-B** [reply]
no, these examples are working as expected.

**Comment by Kris-B** [comment]
I think I could locate the origin of the issue: it seems to be related to layers's properties transitions.
Removing the transitions solves the problems.
I'm now looking to find a solution...

**Reply by Pessimistress** [reply]
It is a known issue that transition does not work well on iOS 15: https://github.com/visgl/deck.gl/issues/6328
